Add explanations (#32)
canonical/ubuntu-packaging-guide: docs/explanation/main-inclusion-review.rst

Main Inclusion Review (MIR)
===========================

.. important::

    Do not confuse the abbreviation :term:`MIR` with the `display server <https://mir-server.io/>`_.

> :term:`Packages <Package>` in :term:`Main` and :term:`Restricted` are officially

Reviewer: ```suggestion
:term:`Packages <Package>` in :term:`main <Main>` and :term:`restricted <Restricted>` are officially
```

maintained, supported and recommended by the :term:`Ubuntu` project.

> :term:`Canonical's <Canonical>` support services applies to these :term:`Packages <Package>`, 
> which include security updates and certain :term:`SLA` grantees when bugs are reported
> and technical support is requested.

Reviewer: ```suggestion
:term:`Canonical's <Canonical>` support services apply to these packages, 
which include security updates and certain :term:`SLA` guarantees when bugs are reported
and technical support is requested.
```

Therefore, special consideration is necessary before adding new :term:`Packages <Package>`
to :term:`Main` or :term:`Restricted`. The :term:`Ubuntu` :term:`MIR Team` reviews
:term:`Packages <Package>` for promotion

- from :term:`Universe` to :term:`Main`
- or from :term:`Multiverse` to :term:`Restricted`.

> This review process is called *Main Inclusion Review (MIR)*.

Reviewer: ```suggestion
This review process is called **Main Inclusion Review (MIR)**.
```

> Apply a package for Main Inclusion Review

Reviewer: This is a slightly odd ordering of words, although I think I know what you meant. I'd suggest either "Main Inclusion Review package application" or "Submit a package for Main Inclusion Review". You could also have "Apply for Main Inclusion Review for a package", although that's slightly longer.

Author: Now that mention it I see it too.

Author: Resolved with 89bec8a88797da66ab13f3414a42b5e33e2670fd

-----------------------------------------

The `Main Inclusion Review Documentation <MainInclusionReviewDocumentation_>`_
by the :term:`MIR Team` provides instructions how to apply a package for
:term:`Main Inclusion Review`. The documentation even contains the instructions
how the application gets reviewed by the :term:`MIR Team`.

.. note::

    The guidelines and review process is constantly evolving. Therefore you should

>     re-read the `Main Inclusion Review Documentation <MainInclusionReviewDocumentation_>`_
>     even if you have applied a package for :term:`Main Inclusion Review` in the past.

Reviewer: ```suggestion
    re-read the `Main Inclusion Review documentation <MainInclusionReviewDocumentation_>`_
    even if you have submitted a package for Main Inclusion Review in the past.
```

>     The `Main Inclusion Review Documentation <MainInclusionReviewDocumentation_>`_ is

Reviewer: ```suggestion
    The Main Inclusion Review documentation is
```

    is also a living document. External contributions, suggestions, discussions or
    questions about the process are always welcome.

> MIR Team weekly Meeting

Reviewer: ```suggestion
MIR team weekly meeting
```

-----------------------

The :term:`MIR Team` holds weekly meetings every Tuesday at 16:30 CET on the
:term:`IRC` Server ``irc.libera.chat`` in the ``#ubuntu-meeting`` channel.
You can follow these `instructions <https://libera.chat/guides/connect>`_ on
how to connect to ``irc.libera.chat``.

> The purpose of the meeting is

Reviewer: ```suggestion
The purpose of the meeting is:
```

- to distribute the workload fairly between the members of the :term:`MIR Team`
- a timely response to reporters of :term:`MIR` applications
- detection and discussion of any current or complex cases

Due to the nature of the :term:`Main Inclusion Review` process there are times
when this meeting is very busy. Usually at the beginning of a new :term:`Ubuntu`

> release not a lot is happening.

Reviewer: This makes it sound like everyone's just chilling lol. I'd recommend rephrasing to something like "Usually, increased activity comes with the period just before a new Ubuntu release, and after that the meetings tend to be quieter." or something like that

Author: lol :D To be honest I have seen MIR meetings in the beginning of a cycle where nothing happened, because there was no MIR request yet.

I agree, I will also suggest changing that in the MIR documentation too ^^

Author: Resolved with 308f8c23277517bf6c5a7c2d901f0fbdbee36ed6
